tests/settings.py

```diff
@@ -142,6 +142,9 @@
 # Disable SECCOMP if set via environment variable.
 FLOW_DOCKER_DISABLE_SECCOMP = strtobool(os.environ.get('RESOLWE_DOCKER_DISABLE_SECCOMP', '0'))
 
+# Ensure all container images follow a specific format.
```

Comment:
I would prefer more specific comment mentioning that `latest` tag is not allowed.

Do we want to check only that the tag is not `latest` or would it be better to allow only tags in form of `x.y.z`?

Reply:
As we've discussed, we have tags like `ubuntu-16.04` so we can't use `x.y.z`.